iroh-net/src/netcheck/reportgen.rs

```diff
@@ -893,22 +780,109 @@ async fn run_probe(
         }
     }
 
-    trace!(probe = ?probe, "probe successful");
+    trace!("probe successful");
     Ok(result)
 }
 
+/// Run a STUN IPv4 or IPv6 probe.
+async fn run_stun_probe(
```

Comment:
maybe add a debug_assert or such to make sure that `derp_addr` and `Probe` match in respect to ipv4/ipv6

Reply:
55499cbd